db/migrate/20151020045115_remove_user_from_neighborhood.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUserFromNeighborhood < ActiveRecord::Migration
+  def change
```

Comment:
@HacksForHugs If neighbors don't have user attributes, this doesn't seem relevant